docs/pages/access-controls/guides/locking.mdx

```diff
@@ -80,6 +81,13 @@ with one of the following options:
   # Created a lock with name "dc7cee9d-fe5e-4534-a90d-db770f0234a1".
   ```
   </TabItem>
+  <TabItem label="Access request">
+  All connections using elevated privileges from the matching access request will be locked.
```

Comment:
I thought these weren't going to work on connections to leaf clusters?  Or was it just that a lock in a leaf cluster couldn't match against the id of a request in a root cluster?  I know we talked about this at some point, but I'm a bit fuzzy on it.

Reply:
It seems like locks targeting access request IDs work fine even in leaf clusters; access request IDs don't have any mapping (they're just UUIDs), so as long as the locks get propagated, there's no reason they shouldn't work.